source/form.html

```diff
@@ -0,0 +1,205 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+
+<head>
+  <meta charset="utf-8">
+  <title>HTML Academy: Кэт энерджи</title>
+</head>
+
+<body class="page__body page-body">
+  <header class="page-header">
+    <a class="page-header__logo">
+      <img class="page-header__logo-image" src="" width="33" height="38" alt="Кэт Энерджи">
+      <img class="page-header__logo-text" src="" width="101" height="18" alt="">
+    </a>
+    <nav class="main-nav">
+      <button class="main-nav__toggle" type="button">
+        <span class="visually-hidden">Открыть меню</span>
+      </button>
+      <div class="main-nav__wrapper">
+        <ul class="main-nav__list site-list">
+          <li class="site-list__item site-list__item--active">
+            <a class="site-list__link" href="index.html">Главная</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="catalog.html">Каталог продукции</a>
+          </li>
+          <li class="site-list__item">
+            <a class="site-list__link" href="form.html">Подбор программы</a>
+          </li>
+        </ul>
+      </div>
+    </nav>
+  </header>
+
+  <main class="page-main">
+    <section class="form">
+      <h1 class="form-title">Подбор программы</h1>
+      <div class="quaestionnaire">
+        <h2 class="visually-hidden">Данные кота</h2>
+        <p class="quaestionnaire__description">Заполните анкету, и мы подберем программу питания для вашего кота</p>
+        <form class="quaestionnaire__form" action="https://echo.htmlacademy.ru/" method="post">
+          <p class="quaestionnaire__wrapper-name">
+            <span class="visually-hidden">Укажите имя кота</span>
+            <label class="quaestionnaire__name" for="quaestionnaire-name-cat">
+              ИМЯ:*
```

Comment:
Лучше писать в стандартном регистре
```suggestion
              Имя:*
```

Reply:
+